src/scss/blocks/_prose.scss

```diff
@@ -50,6 +50,10 @@
     --flow-space: #{get-space('size-2')};
   }
 
+  details + details {
```

Comment:
FWIW, nesting `{% Details %}` seems to result in invalid HTML (at least as per https://github.com/GoogleChrome/web.dev/pull/7648#issuecomment-1088927674). Maybe we shouldn't encourage it until we can find a fix for that?

Reply:
This is not for nesting, but for "next sibling", so that a list of details does not have margin between items:

<img width="957" alt="Screenshot 2022-04-05 at 18 30 27" src="https://user-images.githubusercontent.com/1914261/161802144-45134716-aff1-410e-a118-2f3e67154029.png">

Reply:
Sorry! My lack of CSS knowledge is showing.